tests/tck/features/go/go.IntVid.feature

```diff
@@ -1593,3 +1593,13 @@ Feature: IntegerVid Go  Sentence
     Then the result should be, in any order, with relax comparison:
       | COUNT($-.id) |
       | 4            |
+
+  @skip
```

Comment:
Need not skip this case.

Reply:
It's a bug of storage expression computation of vid, I'm fixing it now.